typescript/sdk/src/metadata/ChainMetadataManager.ts

```diff
@@ -61,14 +58,7 @@ export class ChainMetadataManager<MetaExt = {}> {
    * @throws if chain's name or domain/chain ID collide
    */
   addChain(metadata: ChainMetadata<MetaExt>): void {
-    const parseResult = safeParseChainMetadata(metadata);
-    if (!parseResult.success) {
-      throw new Error(
-        `Invalid chain metadata for ${
-          metadata.name
-        }: ${parseResult.error.format()}`,
-      );
-    }
+    ChainMetadataSchema.parse(metadata);
```

Comment:
was this intentional?

Reply:
It was. Let's let better error msgs surface. IIRC you may have suggested this in the past and I pushed back but after looking closer at it I've come around

Reply:
love it

Reply:
lets goooo